Dockerfile

```diff
@@ -1,34 +1,56 @@
-RUN wget --quiet --output-document - "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | apt-key add -
-RUN echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" > /etc/apt/sources.list.d/kisio-digital.list
-RUN apt update && apt install --yes pkg-config libssl-dev clang libtiff-dev libcurl4-nss-dev proj=${PROJ_VERSION}
+ARG BUILD_DEPENDENCIES
+ARG RUNTIME_DEPENDENCIES
+RUN apt update
+RUN apt install --yes ${BUILD_DEPENDENCIES} ${RUNTIME_DEPENDENCIES}
+RUN wget https://github.com/OSGeo/PROJ/releases/download/${PROJ_VERSION}/proj-${PROJ_VERSION}.tar.gz
+RUN tar -xzvf proj-${PROJ_VERSION}.tar.gz
+RUN mv proj-${PROJ_VERSION} /tmp/proj-src
+WORKDIR /tmp/proj-src
+RUN ./configure --prefix=/usr
+RUN make -j$(nproc)
+# copy all the installation files inside a temporary folder
+# so they are copyable from the following stages of the Docker image
+RUN make DESTDIR=/tmp/proj-build install
 
+FROM debian:stretch as rust-builder
+ARG RUNTIME_DEPENDENCIES
+RUN apt update
+RUN apt install --yes ${RUNTIME_DEPENDENCIES}
+# install 'proj'
+# we copy all the files to the root ('/tmp/proj-build' contains a 'usr/' folder)
+COPY --from=proj-builder /tmp/proj-build /
 WORKDIR /usr/src/app
 COPY . ./
-RUN cargo build --workspace --release \
-	&& mkdir /usr/src/bin && for file in ls ./target/release/*; do if test -f $file -a -x $file; then cp $file /usr/src/bin; fi; done \
-	&& cd .. && rm -rf app
+# install rustup
+RUN apt install --yes curl
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH "/root/.cargo/bin:$PATH"
+# build the project
+RUN cargo build --workspace --release
+RUN mkdir /usr/src/bin && for file in ls ./target/release/*; do if test -f $file -a -x $file; then cp $file /usr/src/bin; fi; done 
 
 FROM debian:stretch-slim
 ARG PROJ_VERSION
-ENV PROJ_DEB "proj_${PROJ_VERSION}_amd64.deb"
-ENV GPG_KEY "C60D758F807A525534C5DFD57B639E3638A8112A"
-VOLUME /app/input
-VOLUME /app/output
-RUN BUILD_DEPENDENCIES="apt-transport-https gnupg2 wget" \
-	&& apt update \
-	&& apt install --yes ${BUILD_DEPENDENCIES} \
-	&& wget --quiet --output-document - "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | apt-key add - \
-	&& echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" > /etc/apt/sources.list.d/kisio-digital.list \
-	&& apt update \
-	&& apt install --yes libtiff-dev libcurl4-nss-dev proj=${PROJ_VERSION} \
-	&& apt purge --yes ${BUILD_DEPENDENCIES} \
-	&& apt autoremove --yes \
-	&& rm -rf /var/lib/apt/lists/*
+ARG RUNTIME_DEPENDENCIES
+# install 'proj'
+# We copy all the files to the root ('/tmp/proj-build' contains a 'usr/' folder)
+COPY --from=proj-builder /tmp/proj-build /
+RUN apt update \
+    && apt install --yes ${RUNTIME_DEPENDENCIES} \
+    && apt autoremove --yes \
+    && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /usr/src/bin/* /usr/local/bin/
```

Comment:
```suggestion
COPY --from=rust-builder /usr/src/bin/* /usr/local/bin/
```

Reply:
Fixed in 53ce6c1a24f668febcc59c5fc76660b05277acbe. Thanks